app/experimenter/static/js/tests/DesignForm.test.js

```diff
@@ -0,0 +1,267 @@
+import React from "react";
```

Comment:
Okay so I just had a long conversation with @rehandalal about this, and I think I understand what's going on here.

What @rehandalal is suggesting is that, bare minimum, we start by testing the actual UI that users interact with.  So in that context that means specifically AddonForm, PrefForm, GenericForm, RolloutForm.

So I think what we want is four test files, one for each of those, that only sets up and instantiates DesignForm in one of those modes, and then tests all the interactions with it that we care about, just like you've done here.

So rename this file to AddonForm.test.js, move the render test you had in `Branch.test.js` into here, and then we can land this as the AddonForm tests and do the other 3 as follow ups.